src/drivers/bitbucket_cloud.js

```diff
@@ -254,14 +254,14 @@ class BitbucketCloud {
     repo.username = user;
 
     const command = `
-    git config --unset user.name && \\
-    git config --unset user.email && \\
-    git config --unset push.default && \\
+    git config --unset user.name;
+    git config --unset user.email;
+    git config --unset push.default;
     git config --unset http.http://${this.repo
       .replace('https://', '')
-      .replace('.git', '')}.proxy && \\
-    git config user.name "${userName || this.userName}" && \\
-    git config user.email "${userEmail || this.userEmail}" && \\
+      .replace('.git', '')}.proxy;
+    git config user.name "${userName || this.userName}" &&
+    git config user.email "${userEmail || this.userEmail}" &&
```

Comment:
this was an accidental change, right?

```suggestion
    git config user.name "${userName || this.userName}" && \\
    git config user.email "${userEmail || this.userEmail}" && \\
```